Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -68,7 +68,7 @@ public final class SwiftPMWorkspace {
   /// - `packageGraph`
   /// - `fileToTarget`
   /// - `sourceDirToTarget`
-  let queue: DispatchQueue = .init(label: "SwiftPMWorkspace.queue", qos: .utility)
+  let queue: DispatchQueue = .init(label: "SwiftPMWorkspace.queue", qos: .userInitiated)
```

Comment:
We don't generally want reloading the package to be run at high priority, or it risks hurting performance in the rest of the system.  We also don't need this for the current change, because
the `self.queue.sync` in `fileHandlingCapability` will cause priority donation if the calling code is running at higher priority than utility.

Reply:
Oh, I didn't know that GCD automatically increases the priority if the caller has a priority higher than the called queue. That’s really good to know 👍 It was the only reason why I increased priority here. I’ll revert it.